catalog/spatial/spatial-commands/src/test/java/org/codice/ddf/commands/spatial/gazetteer/GazetteerBuildSuggesterIndexCommandTest.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.commands.spatial.gazetteer;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.not;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doThrow;
+
+import ddf.catalog.CatalogFramework;
+import ddf.catalog.filter.FilterBuilder;
+import ddf.catalog.source.SourceUnavailableException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Answers;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GazetteerBuildSuggesterIndexCommandTest {
+  @Mock private CatalogFramework catalogFramework;
+
+  @Mock(answer = Answers.RETURNS_DEEP_STUBS)
+  private FilterBuilder filterBuilder;
+
+  private ConsoleInterceptor consoleInterceptor;
+
+  private GazetteerBuildSuggesterIndexCommand command;
+
+  @Before
+  public void setUp() {
+    consoleInterceptor = new ConsoleInterceptor();
+    consoleInterceptor.interceptSystemOut();
+    command = new GazetteerBuildSuggesterIndexCommand();
+    command.setFilterBuilder(filterBuilder);
+    command.setCatalogFramework(catalogFramework);
+  }
+
+  @After
+  public void tearDown() {
+    consoleInterceptor.resetSystemOut();
+  }
+
+  @Test
+  public void successfulRunPrintsNoErrors() {
+    command.executeWithSubject();
+
+    final String output = consoleInterceptor.getOutput();
+    assertThat(output, not(containsString("Error:")));
+  }
+
+  @Test
+  public void unsuccessfulRunPrintsErrors() throws Exception {
```

Comment:
The alternative is `throws UnsupportedQueryException, SourceUnavailableException, FederationException`. I think `throws Exception` is acceptable for tests because we don't really care what exceptions the test method might throw (as the test will fail if any exception is thrown).

Reply:
I agree, `throws Exception` is good enough for tests but not for production code